### ISSUE TITLE
[MU4 Issue] Backup file not created when saving over an imported MU3 file

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16481]
**Describe the bug**
Normally if you open a score in MuseScore, modify it, then save it, a backup is created - thus allowing you to restore the file to its state from when you first opened it.  Unfortunately, this does not happen for imported MU3 files, meaning if you save over it (ignoring the warning, as people often to), there is no recovering the previous version.

**To Reproduce**
1. Open an MU3 file
2. Save (keeping original folder/filename, ignoring the overwrite warning)
3. Examine the .mscbackup folder
4. See error: no backup is created

**Expected behavior**
A backup should be created in that folder, same as if you had started by opening an MU4 file

**Platform information**
 - OS: Linux

**Additional context**
There is considerable pain being expressed in the community regarding people overwriting their MU3 files with MU4 and not realizing they can't go back.  While we can't really do anything about that in general, we can certainly make sure we're creating a backup on that first save, which we do normally anyhow but for some reason aren't doing it here when it's needed most.

**Comment by Jojo-Schmitz** [comment]
IMHO this is not a bug at all. Mu3 and Mu2 behaved the very same, and beina "Save As" this is IMHO by design...

But it'd indeed make for a nice new feature and resolve a major pain point.

**Comment by MarcSabatella** [comment]
Good point, and I'd actually say, *any* time you "save as" and overwrite an existing file (whether the result of import or not), that would potentially be a fine time to trigger a backup.

**Comment by Jojo-Schmitz** [comment]
Yes.

**Comment by bakajikara** [comment]
I think this is already been fixed by #25458.

**Comment by cbjeukendrup** [comment]
I can confirm, thanks!
